apps/api/src/billing/lib/batch-signing-client/batch-signing-client.service.ts

```diff
@@ -288,4 +296,23 @@ export class BatchSigningClientService {
   private async simulate(messages: readonly EncodeObject[], memo: string) {
     return (await this.clientAsPromised).simulate(await this.getCachedFirstAddress(), messages, memo);
   }
+
+  private async getFreshAccountInfo(): Promise<ShortAccountInfo> {
```

Comment:
seems like this function duplicates the implementation of `getCachedAccountInfo`. if so, it needs to be used there, right?